packages/vitest/src/integrations/snapshot/port/utils.ts

```diff
@@ -161,17 +161,17 @@ export async function saveSnapshotFile(
 
 export function prepareExpected(expected?: string) {
   function findStartIndent() {
-    const match = /^( +)}\s+$/m.exec(expected || '')
+    const match = /^( +)[}|"]/m.exec(expected || '')
```

Comment:
I'm quite unsure what this was supposed to do earlier:


`^( +)}\s+$`

- `^` From the beginning
- `( +)` Capture at least one space...
- `}` ... before } character
- `\s+$` ... only when whole expression ends with whitespaces

... right?

Should this have been without `^` instead?

Reply:
Just realized there's the `m` modifier for multilines. Applied required changes for this.